src/tests/api-tests.ts

```diff
@@ -3120,6 +3120,60 @@ describe('api tests', () => {
     expect(fetchBlockByHeight.status).toBe(200);
     expect(fetchBlockByHeight.type).toBe('application/json');
     expect(JSON.parse(fetchBlockByHeight.text)).toEqual(expectedResp);
+
+    const fetchBlockByBurnBlockHeight = await supertest(api.server).get(
+      `/extended/v1/block/by_burn_block_height/${block.burn_block_height}`
+    );
+    expect(fetchBlockByBurnBlockHeight.status).toBe(200);
+    expect(fetchBlockByBurnBlockHeight.type).toBe('application/json');
+    expect(JSON.parse(fetchBlockByBurnBlockHeight.text)).toEqual(expectedResp);
+
+    const fetchBlockByInvalidBurnBlockHeight1 = await supertest(api.server).get(
+      `/extended/v1/block/by_burn_block_height/999`
+    );
+    expect(fetchBlockByInvalidBurnBlockHeight1.status).toBe(404);
+    expect(fetchBlockByInvalidBurnBlockHeight1.type).toBe('application/json');
+    const expectedResp1 = {
+      error: 'cannot find block by height 0',
```

Comment:
CI failure showing
```
    - Expected  - 1
    + Received  + 1

      Object {
    -   "error": "cannot find block by height 0",
    +   "error": "cannot find block by height 999",
      }
```

Reply:
How can I run these locally? npm run test fails with 
```
Waiting for RPC connection to core node..
FetchError: request to http://127.0.0.1:20443/v2/info failed, ...
```

Reply:
`npm run test:integration`

To ensure docker cache isn't causing issues, you can run this:
```
npm run devenv:build
npm run test:integration
```

Reply:
Thank you. I should have read the readme.